### DISCUSSION TITLE
Internationalization plans for OpenSign

### DISCUSSION THREAD

**Discussion by gustavobhz** [OpenSignLabs/OpenSign#653] [General]
Hi there,

I've been following the development of OpenSign and I'm quite impressed with the project's progress so far.

I wanted to inquire if there are any plans or discussions regarding the implementation of internationalization (i18n) for OpenSign. As the project gains traction, making it accessible to users from diverse linguistic backgrounds could significantly enhance its reach and impact. If there are no ongoing efforts in this direction, I'd be interested in contributing to such an initiative. I believe enabling multi-language support could greatly benefit the project.

**Comment by Zathiel** [comment]
Hi @gustavobhz ,

Hope you are well !

We are currently working with internationalization.

Wich langages do you need the most

Regards

**Reply by gustavobhz** [reply]
Hi @Zathiel,

I'm doing well, thank you!

I would like to contribute to the internationalization efforts by adding Brazilian Portuguese to the project. Could you also let me know which branch the internationalization work is being done on?

**Reply by Zathiel** [reply]
Its currently on the way  i will keep u in touch when everything is setup 💪🏻

**Reply by TSP-Dev** [reply]
Hi @Zathiel,
How is the progress of this feature?
Maybe we can contribute for spanish.
which branch are you workin on?

**Reply by Zathiel** [reply]
Hi,

Still ongoing I will let you know when the branch the PR is created here as we are working on a other repository with someone from the community 

Regards

**Reply by zagvier** [reply]
Hi @Zathiel,

Maybe we can contribute for french.

**Reply by zyriab** [reply]
Hello, I'd like to contribute to the French i18n efforts, my customers usually don't speak English and I'd love to give them a UI they can understand. 😄
